LondonBritishLibrary/orient/BLorient8819.xml

```diff
@@ -0,0 +1,279 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient8819" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Discourses on the Passion for Holy Week, Salām to Jesus Christ, Salām to the Virgin Mary</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 8819</idno>
+                        <altIdentifier><idno>Or. 8819</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 57</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="4r" to="78r"/> 
+                            <title ref="LIT6193HomiliaryPassions">Discourses on the Passion of our Lord Jesus Christ for Holy Week</title>  
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="4r"/> 
+                                <title ref="LIT6193HomiliaryPassions#Introduction"/>
+                                <note>Rhyming</note>
+                                <incipit xml:lang="gez">በስመ፡ እግዚአብሔር፡ ሥሉስ፡ ዘእንበለ፡ ቀዲም፡ ወተድኅሮ። በግጻዌሁ፡ ይትረአይ፡ ዘአስተዋሐደ፡ ምክሮ። በጠፈረ፡ በረድ፡ ወነድ፡
```

Comment:
Perhaps you can use code to visualize the rhyme and show that it does not coincide with the actual line division, as explained here (https://betamasaheft.eu/Guidelines/?id=text-encoding)

Reply:
`<l n="1"></l>` or `<ab></ab>` is not allowed inside `<incipit></incipit>`.

Do you want me to highlight the line breaks in the work record?